config/configs_basic.yaml

```diff
@@ -66,7 +66,7 @@ timeseries_analyzer:
   auto_detection: True
   id_col: 'ifa'
   tz_offset: 'local'
-  inspection: False
+  inspection: True
```

Comment:
inspection can remain "False" in configs_basic.yaml
in PR #254 also it is False.
in configs_full.yaml it is set to True